Imgsmall.java

```diff
@@ -0,0 +1,34 @@
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.awt.image.RenderedImage;
+import java.io.File;
+import java.net.URL;
+
+public class Imgsmall {
+  public static void main(String[] args) throws Exception{
+    //get url object
+    URL url = new URL(args[0]);
```

Comment:
Note that if URL is malformed you will get a MalformedURLException (MFU). You should protect against user sending you a bad URL (eg - htp://google.com (note htp instead of http etc). so you should wrap the code between try catch and when a MFU is thrown you tell the user to check their URL.